pinot-spi/pom.xml

```diff
@@ -149,6 +149,20 @@
       <groupId>org.reflections</groupId>
       <artifactId>reflections</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Please also apply the same pom changes for https://github.com/apache/pinot/blob/master/pinot-connectors/prestodb-pinot-dependencies/pinot-spi-jdk8/pom.xml

Also other pinot jdk8 modules if any: https://github.com/apache/pinot/tree/master/pinot-connectors/prestodb-pinot-dependencies